docs/network-setup.md

```diff
@@ -58,8 +58,7 @@ curl -X PUT \
     }'
 ```
 
-Alternatively, if you are using firectl, add `--tap-
-device=veth0/AA:FC:00:00:00:01` to your command line.
+Alternatively, if you are using firectl, add `--tap-device=tap0/AA:FC:00:00:00:01` to your command line.
```

Comment:
Please add a newline after `--tap-` as it was earlier.

Reply:
@aghecenco sure but why it needs newline? the key is --tap-device=

Reply:
We are keeping all lines less than 80 characters. That's why you need the newline. You could also have it like:
```
Alternatively, if you are using firectl, add 
`--tap-device=tap0/AA:FC:00:00:00:01` to your command line. 
```

Reply:
@andreeaflorescu thanks, I just fixed it.